openlibrary/macros/WorkCarousel.html

```diff
@@ -0,0 +1,5 @@
+$def with(work)
+
+$:render_template("home/custom_ia_carousel", title="You may also like:", key="related-subjects-carousel", work_id=work.key, _type="subjects", limit=42)
+
+$:render_template("home/custom_ia_carousel", title="More by this author:", key="related-authors-carousel", work_id=work.key, _type="authors", limit=42)
```

Comment:
Same here, if possible I think "More works by H. G. Wells" is a wee bit nicer/more transparent. Either way, probably drop the ':'; the fact that it's large bold text already communicates that it is a heading, so the colon feels a little redundant.